client/client.go

```diff
@@ -271,6 +271,9 @@ func (o *OpkClient) oidcAuth(
 	return pkt, nil
 }
 
+// Refreshes the ID token and access token for the OIDC provider using the refresh token.
```

Comment:
GoDoc function comments start with the name of the function, checkout the GoDoc guide  https://go.dev/doc/comment

Reply:
Corrected the format.